css/style.css

```diff
@@ -550,17 +550,20 @@ figcaption details p {
 }
 
 .floe .floe-image {
-    width: 325px;
-    height: 325px;
+    width: 270px;
+    display: block;
+    height: 280px;
```

Comment:
![Screenshot from 2020-03-11 22-59-55](https://user-images.githubusercontent.com/34926285/76445847-14c35a00-63ec-11ea-9f5b-2cd548e1f592.png)

This apparently doesn't fix the images.